### PR TITLE
Latest swedish translations

### PR DESCRIPTION
**Pull request by reynders**

### CODE REVIEW

File: frontend/packages/citizen-frontend/src/localization/sv.tsx [hunk 1]
```diff
@@ -172,11 +172,12 @@ const sv: Translations = {
                   Du kan ansöka om plats i småbarnspedagogisk verksamhet året
                   om. Ansökningen bör lämnas in senast fyra månader före behovet
                   av verksamheten börjar. Om behovet börjar med kortare varsel
-                  bör du ansöka om plats senast två veckor före.
+                  pga. sysselsättning eller studier bör du ansöka om plats
+                  senast två veckor före.
                 </P>
                 <P>
                   Du får ett skriftligt beslut om platsen. Beslutet delges i
-                  tjänsten{' '}
```
Comment (kustaahuhtala) [inline frontend/packages/citizen-frontend/src/localization/sv.tsx:179]: Just to be on the safe side: there will be whitespace rendered between "tjänsten" and anchor element?

**Review by massproduction — APPROVED** [review]

File: frontend/packages/citizen-frontend/src/localization/sv.tsx [hunk 2]
```diff
@@ -702,7 +687,7 @@ const sv: Translations = {
         units: {
           title: 'Ansökningsönskemål',
           startDateMissing:
-            'För att välja önskade enheter, välj det första önskade startdatumet i avsnittet om "Servicebehov"',
+            'För att kunna välja önskade enheter, välj först det önskade startdatumet i avsnittet om "Servicebehov"',
```
Comment (kustaahuhtala) [inline frontend/packages/citizen-frontend/src/localization/sv.tsx:690]: Do we actually have a section called "Servicebehov"? (I think we used to, but not anymore)

File: frontend/packages/citizen-frontend/src/localization/sv.tsx [hunk 3]
```diff
@@ -897,13 +875,13 @@ const sv: Translations = {
         secondGuardianInfo:
           'Den andra vårdnadshavarens information hämtas automatiskt från befolkningsinformationssystemet.',
         secondGuardianNotFound:
-          'Baserat på information från VTJ har barnet ingen andra vårdnadshavare',
+          'Baserat på information från befolknngsdata har barnet ingen andra vårdnadshavare',
```
Comment (kustaahuhtala) [inline frontend/packages/citizen-frontend/src/localization/sv.tsx:878]: I guess exactly correct translation for väestötietojärjestelmä would be Befolkningsdatasystemet. But I guess it doesn't matter here.

**Review by kustaahuhtala — APPROVED** [review]
Left some comments, but no show stoppers. Maybe worth checking them out in any case.

**Pull request merged**